### PR TITLE
[core] Some cleaning related to cmake messages

### PR DESCRIPTION
**Pull request by josep-m-carbonell**
Adding STATUS to some messages and erasing some "special" lines

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -209,7 +209,7 @@ endif(CMAKE_BUILD_TYPE MATCHES FullDebug)
 
 ##*****************************
 #find and include OpenMP if possible
-message(KRATOS_EXCLUDE_OPENMP ${KRATOS_EXCLUDE_OPENMP})
+#message(STATUS KRATOS_EXCLUDE_OPENMP : ${KRATOS_EXCLUDE_OPENMP})
```
Comment (philbucher) [inline CMakeLists.txt]: please remove commented code
this confuses if one later reads the code

+1 for the removal of this line

Comment (josep-m-carbonell) [reply CMakeLists.txt]: No problem, I think this file must be reviewed deeply later on. Usually this variable ${KRATOS_EXCLUDE_OPENMP} is empty and prints nothing. I will delete this line.

**Review by RiccardoRossi — APPROVED** [review]

**Comment by josep-m-carbonell** [comment]
@roigcarlo  can you restart appveyor please?

**Comment by roigcarlo** [comment]
Done

**Comment by josep-m-carbonell** [comment]
@roigcarlo something is happening with appveyor.

**Comment by roigcarlo** [comment]
All Ci did work now.

**Pull request merged**